cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -3705,6 +3705,27 @@ search = `cs_kaspersky` ProductName=KES*  \
 | outputlookup cs_kaspersky_inventory_v2
 action.cyences_notable_event_action.products = Kaspersky
 
+[Device Inventory - Splunk Internal - V2]
```

Comment:
This should be similar to Lansweeper. Get the latest things only, and we can  delete old stuff.